dotnet/samples/KernelSyntaxExamples/Example65_HandlebarsPlanner.cs

```
@@ -127,7 +127,9 @@ private static async Task PlanNotPossibleSampleAsync(bool shouldPrintPrompt = fa
             // Load additional plugins to enable planner but not enough for the given goal.
             await RunSampleAsync("Send Mary an email with the list of meetings I have scheduled today.", shouldPrintPrompt, "SummarizePlugin");
         }
-        catch (KernelException e)
+        catch (KernelException ex) when (
+            ex.Message.Contains("The plan references hallucinated helpers", StringComparison.CurrentCultureIgnoreCase)
+            || ex.Message.Contains("Unable to create plan for goal with available functions", StringComparison.CurrentCultureIgnoreCase))
```

We just swallow these errors and do nothing when we get them?